packages/webdriverio/src/commands/element/getCSSProperty.ts

```diff
@@ -64,50 +66,137 @@ import { parseCSS } from '../../utils/index.js'
         // }
     })
  * </example>
- *
  * @alias element.getCSSProperty
- * @param  {string}      cssProperty css property name
- * @return {CSSProperty}             The specified css of the element
+ * @param  {string}        cssProperty   css property name
+ * @param  {PseudoElement} pseudoElement css pseudo element
+ * @return {CSSProperty}                 The specified css of the element
  *
  */
 export async function getCSSProperty (
     this: WebdriverIO.Element,
-    cssProperty: string
+    cssProperty: string,
+    pseudoElement?: PseudoElement,
+) {
+    if (cssShorthandProps.isShorthand(cssProperty)) {
+        const cssValue = await getShorthandPropertyCSSValue.call(
+            this,
+            {
+                cssProperty,
+                pseudoElement,
+            }
+        )
+
+        return parseCSS(cssValue, cssProperty)
+    }
+
+    const cssValue = await getPropertyCSSValue.call(
+        this,
+        {
+            cssProperty,
+            pseudoElement,
+        }
+    )
+
+    return parseCSS(cssValue, cssProperty)
```

Comment:
We can simplify this to:
```suggestion
    const getCSSProperty = cssShorthandProps.isShorthand(cssProperty)
        ? getShorthandPropertyCSSValue
        : getPropertyCSSValue

    const cssValue = await getCSSProperty.call(
        this,
        {
            cssProperty,
            pseudoElement,
        }
    )

    return parseCSS(cssValue, cssProperty)
```